src/pages/index.html

```diff
@@ -0,0 +1,504 @@
+{% extends "layout.html" %} {% set title = 'Starct' %} {% block content %}
+
+<section class="header-main" id="home">
+    <div class="container">
+        <div class="row align-items-center justify-content-center">
+            <div class="col-lg-7 col-xl-8">
+                <div class="main-text">
+                    <h2>
+                        <span class="text-decoration-underline">Secure</span>
+                        your business, Secure your
+                        <span class="text-decoration-underline">future</span>
+                    </h2>
+                    <p class="text-center">
+                        Data analysis software is a type of software tool used
+                        for data analysis and reporting. It is designed to help
+                        businesses, organizations, and individuals process,
+                        visualize,
+                    </p>
+                </div>
+                <div
+                    class="d-flex justify-content-center align-items-center btn-primary-main"
+                >
+                    <button class="btn-head text-center">
+                        Get Started Free
+                    </button>
```

Comment:
@shaban011147 @M-Khairy-B It looks very bad on responsive, rework the responsive.

Reply:
Done